Storage/Rust/RustPlaces.swift

```diff
@@ -161,6 +161,12 @@ public class RustPlaces {
         }
     }
 
+    public func getRecentBookmarks() -> Deferred<Maybe<[BookmarkItem]>> {
```

Comment:
Put a `limit: UInt = 20` argument in here and pass it on to the FFI call so that the caller can override and make a determination about how many to get.